integration/testdata/update-dependency-from-metadata/multiple-constraints-example/expected.toml

```diff
@@ -43,9 +43,9 @@ api = "0.2"
     licenses = ["some-license"]
     name = "Some Dependency"
     purl = "some-purl"
-    checksum = "sha256:some-sha"
+    sha256 = "some-sha"
     source = "some-source"
-    source-checksum = "sha256:some-source-sha"
+    source_sha256 = "some-source-sha"
```

Comment:
a test case like this with both checksum and sha256 fields is meant to show that when a version isn't added or deleted, it's un-modifed. As in, `jam` isn't going to modify the `sha256` field on any version that isn't being added.